frontend/web/components/App.js

```diff
@@ -596,6 +599,17 @@ const App = class extends Component {
                           >
                             Compare
                           </NavSubLink>
+                          {environmentMetricsEnabled && (
+                            <NavSubLink
+                              icon={gitCompare}
+                              to=''
+                              id='reporting-link'
+                              disabled
+                              tooltip='Coming soon'
+                            >
+                              Reporting
+                            </NavSubLink>
+                          )}
```

Comment:
This part was discussed but not 100% validated. So if it is in this version maybe it should have its own flag